.github/workflows/component_canaries_prune.yml

```diff
@@ -19,8 +22,9 @@ permissions:
   id-token: write
 
 jobs:
-  canaries-prune:
-    if: ${{ always() }}
+
+  canaries-prune-linux:
+    if: ${{ inputs.PLATFORM == 'linux' || inputs.PLATFORM == '' }}
```

Comment:
Maybe it's better to have an explicit option 'all'

Reply:
Agree. Done in https://github.com/newrelic/infrastructure-agent/pull/1794/commits/3b6ea17d7d4da595b4b91257120a578eae92bf13